source/Engine.cpp

```diff
@@ -720,6 +720,22 @@ void Engine::Step(bool isActive)
 		info.SetString("navigation mode", "Navigation:");
 		info.SetString("destination", "no destination");
 	}
+	//display x,y coordinates
+	if (flagship)
+	{
+		//Point position = player.GetSystem()->Position(); // Galactic Cooordinates.
+		Point position = flagship->Position();
+		int d = position.Length();
+		int x = position.X();
+		int y = position.Y();
+		info.SetString("dhead", "D:");
+		info.SetString("xhead", "X:");
+		info.SetString("yhead", "Y:");
```

Comment:
this seems to be constant, cant we just define it in interfaces.txt

Reply:
In fact interfaces would make it more customizable, so please move it there.

Reply:
Is it just as simple as moving it, or am I going to need some extra code somewhere to tell it to load that from the interfaces file?

Reply:
you should be able to just define them in the interface file.

Reply:
This has also all been removed.